app/tools/jsondb.py

```diff
@@ -5,74 +5,63 @@ class Database:
     def __init__(self, name_space):
         self.name_space = name_space
         self.files = {}
-        self.create()
+        self._load_file(name_space)
 
     def create(self):
-        if self.name_space in self.files:
-            return False
-        else:
-            self.files[self.name_space] = []
-            self._save_file(self.name_space)
-            return True
+        self.files[self.name_space] = []
+        self._save_file()
+        return True
 
     def add(self, name_space, fields):
-        if name_space in fields:
-            raise OSError("Namespace already exist!")
-
-        records = self.files[name_space]
+        records = self.files[self.name_space]
         if len(records) == 0:
             new_id = 1
         else:
             new_id = records[-1]['id'] + 1
 
         fields['id'] = new_id
         records.append(fields)
-        self._save_file(name_space)
+        self._save_file()
         return new_id
 
     def delete(self, name_space, record_id):
-        if name_space in self.files:
-            return False
-
-        records = self.files(name_space)
+        records = self.files[self.name_space]
         for record in records:
             if record['id'] == record_id:
-                record.remove(record)
-                self._save_file(name_space)
+                records.remove(record)
+                self._save_file()
                 return True
-        False
-
-    def get_all(self, name_space):
-        if name_space not in self.files:
-            return []
-
-        return self.files[name_space]
+        return False
 
-    def get_by_id(self, name_space, record_id):
-        if name_space not in self.files:
-            return []
+    def get_all(self):
+        return self.files[self.name_space]
 
-        for record in self.files[name_space]:
+    def get_by_id(self, record_id):
+        for record in self.files[self.name_space]:
             if record['id'] == record_id:
                 return [record]
 
         return []
 
-    def update(self, name_space, fields):
-        if name_space not in self.files:
-            return False
-
-        records = self.files[name_space]
+    def update(self, fields):
+        records = self.files[self.name_space]
         for record in records:
             if record['id'] == fields['id']:
                 record.update(fields)
-                self._save_file(name_space)
+                self._save_file()
                 return True
 
         return False
 
+    def _load_file(self, name_space):
```

Comment:
po co tutaj name_space

Reply:
pamiętaj, nie twórz długich metod. Zgodnie z zasadą Single-responsibility principle metoda powinna robić jedną rzecz, tutaj metoda określa nazwę pliku (to jest 1 rzecz) oraz zapisuje plik (2 rzecz)